app/api/2fa/verify/route.ts

```diff
@@ -0,0 +1,65 @@
+import type { NextRequest } from "next/server";
```

Comment:
I think we would need rate limiting on verification endpoints, otherwise an attacker could brute force 2FA codes.

Might also be a good idea on /disable, /setup, /check and /status.